docs/apache-airflow-providers-google/operators/cloud/looker.rst

```diff
@@ -0,0 +1,71 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
```

Comment:
The license header is invalid. Several tabs are missing.

Reply:
Done! ✅